RecoTracker/IterativeTracking/python/DetachedTripletStep_cff.py

```diff
@@ -252,15 +254,20 @@
 detachedTripletStep = ClassifierMerger.clone()
 detachedTripletStep.inputClassifiers=['detachedTripletStepClassifier1','detachedTripletStepClassifier2']
 
+trackingPhase1.toReplaceWith(detachedTripletStep, detachedTripletStepClassifier1.clone(
+     mva = dict(GBRForestLabel = 'MVASelectorDetachedTripletStep_Phase1'),
+     src = 'detachedTripletStepTracks',
+     qualityCuts = [-0.2,0.3,0.8]
+))
+(trackingPhase1 & fastSim).toModify(detachedTripletStep,vertices = "firstStepPrimaryVerticesBeforeMixing")
```

Comment:
this is not needed and may be a bit confusing re pp_on_AA and highBeta below: the fastsim value is inherited from L245 and this line can be removed

Reply:
Removed redundant line